cves/kernel/CVE-2018-20961.yml

```diff
@@ -454,7 +456,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: This is a coding lapse. The developer forgot to check if resource was already free
 CWE_instructions: |
   Please go to http://cwe.mitre.org and find the most specific, appropriate CWE
   entry that describes your vulnerability. We recommend going to
```

Comment:
Add to Upvotes (since not changed in commit): I give this a 2

Reply:
I give this 2 upvotes as well

Reply:
I give this 3 upvotes